WAW.API/Chat/Domain/Services/IChatRoomService.cs

```diff
@@ -0,0 +1,11 @@
+using WAW.API.Chat.Domain.Models;
+using WAW.API.Chat.Domain.Services.Communication;
+
+namespace WAW.API.Chat.Domain.Services;
+
+public interface IChatRoomService {
+  Task<IEnumerable<ChatRoom>> ListAll();
+  Task<ChatRoomResponse> Create(ChatRoom chatRoom);
+  Task<ChatRoomResponse> Update(long id, ChatRoom chatRoom);
+  Task<ChatRoomResponse> Delete(long id);
+}
```

Comment:
I think this should include a method to retrieve all the messages that belong to this chat room. The controller should also have this.

Reply:
I think the same but that method maybe can be implemented in the next controllers with relationships `ChatRoomMessageController`

Reply:
Awesome, makes sense.